x-pack/plugins/infra/common/log_entry/log_entry_cursor.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+import { decodeOrThrow } from '../runtime_types';
+
+export const logEntryCursorRT = rt.type({
```

Comment:
Moving this out of the http directory to make it usable in search strategies as well.